astro/src/styles/bootstrap.scss

```diff
@@ -95,6 +95,7 @@ $theme-colors-border-subtle-dark: map-merge($theme-colors-border-subtle-dark, $c
 @import "bootstrap/scss/navbar";
 // @import "bootstrap/scss/card";
 @import "bootstrap/scss/list-group";
+@import "bootstrap/scss/card";
```

Comment:
This is picky, but can you use the one on line 96 and delete this line?

That maintains the order of imports the same as default bootstrap, which is really important in terms of keeping the look-and-feel of bootstrap to match what we expect.